components/multiselect/multiselect_list.jsx

```diff
@@ -160,6 +160,7 @@ export default class MultiSelectList extends React.Component {
             <div className='more-modal__list'>
                 <div
                     ref='list'
+                    id='userAddList'
```

Comment:
This component is generic, not only for user selection.  I recommend to change this, maybe something like: `multiSelectList`